src/templates/kit/prop-types.js

```diff
@@ -0,0 +1,23 @@
+import PropTypes from 'prop-types'
+
+export const ScreenPropsType = PropTypes.shape({
```

Comment:
In `prop-types` we've used the `Type` suffix only, rather than `PropsType`. What do you think about changing this to just `ScreenType`?

Reply:
It’s the `Type` suffix added to `screenProps`. I know it’s confusing :laughing: